denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/functions/mod_notifier.dsc

```diff
@@ -42,6 +42,20 @@ mod_notify_action:
     - else:
       - run mod_message_staff def:<[message]>
 
+# -- Notify online staff about reports.
+mod_notify_report:
+  type: task
+  debug: true
+  definitions: reporter|uuid|reason|server
+  script:
+    - define reporter <[reporter].as_player.name>
+    - define name <[uuid].as_player.name>
+    - define message "<&6>[<&a><&l><[server]><&6>] <&e><[reporter]> <&a>has reported <&c><[name]> <&a>for <&6><[reason]>."
+    - if <bungee.list_servers.size||0> > 1:
```

Comment:
This will never return invalid.
optimally, it's better to check if a list is empty or not as opposed to comparing a list's size to a number
I'd recommend ` if <bungee.list_servers.is_empty>`, but this list specifically shouldn't ever be empty
if you meant that you wanted to check if servers were connected or not, I'd recommend `<bungee.connected>` - else-wise I'd recommend removing this section entirely and only doing one or the other.
```suggestion
    - if <bungee.connected>:
```